src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,27 @@
+import { useContext, useEffect } from 'react';
+import { DispatchContext, StatesContext } from '../context/Store';
+import classNames from 'classnames';
+
+export const ErrorNotification: React.FC = () => {
+  const { errorMessage } = useContext(StatesContext);
+  const dispatch = useContext(DispatchContext);
+
+  useEffect(() => {
+    if (errorMessage) {
+      setTimeout(() => dispatch({ type: 'showError', payload: null }), 3000);
+    }
+  });
```

Comment:
Here you should also include the dependency array with `errorMessage`and `dispatch`